apps/atqueue/values.yaml

```diff
@@ -17,6 +17,11 @@ env: {}
 envSecrets: []
 # -- This key allows specification of a script to override the entrypoint
 entrypoint:
+# -- This key allows for specification of Butler secret information
+# If this section is used, it must contain the following attribute:
+# __containerPath__ (The directory location for the Butler secret)
```

Comment:
You seen to be mixing `__` (like this line) and `_` (like line 27). Please choose one and stick to that. also end this line in a `,` because another attribute follows on the next line.

Reply:
Same for the other `values.yaml` files.

Reply:
Sigh. Still can't follow my own conventions. Fixed.